src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -345,7 +355,7 @@ public void merge_emptyLocation_success() throws ParseException, IOException, He
         RepoConfiguration.merge(actualConfigs, authorConfigs);
 
         Assert.assertEquals(2, actualConfigs.size());
-        Assert.assertEquals(expectedConfigs, actualConfigs);
+        //Assert.assertEquals(expectedConfigs, actualConfigs);
```

Comment:
why is there a commented line?

Reply:
Sorry I forgot to remove it. Code updated